pangeo-notebook/packages.txt

```diff
@@ -131,7 +132,7 @@ earthaccess==0.5.3
 eccodes==2.31.0
 entrypoints==0.4
 eofs==1.4.0
-erddapy==2.2.0
```

Comment:
significant drop here. i'm not familiar with this library...

Reply:
I managed to force `erddapy>=2.2.0` in the environment.yml file and lock it with `conda-lock=2.3` locally. Let's try running `/condalock` again?

Reply:
Yeah, still not working. Maybe it's because we're still using `conda-lock=1.4` here?

https://github.com/pangeo-data/pangeo-docker-images/blob/48c911ca7f917d9b480fa9f2cfd8e4bf0c45e75d/environment-condalock.yml#L5-L6

Let me open another PR to bump up the conda-lock version.

Reply:
Hmm, erddapy is still locked to 0.7.2. Maybe we should manually set a minimum pin to `erddapy>=2.2.20` here

https://github.com/pangeo-data/pangeo-docker-images/blob/f6c380c948de63fe4bec01895062291cf7dcacb3/pangeo-notebook/environment.yml#L21

Reply:
I think argopy is actually the issue: 

`conda-tree -n testpy11 whoneeds erddapy -t --small` 
```
erddapy==0.7.2
  └─ argopy 0.1.6 [required: >=0.5,<0.8.0]
```

`conda search argopy --info`
NOTE: More recent versions of argopy pin `- gsw <=3.4.0` 

But we don't have `gsw` pinned and this is a decision point for conda-lock: Either 1) use an new version of `gsw` and pin argopy/erddapy to older versions. or (2) pin old gsw and use newer erddapy/argopy.

Reply:
opened https://github.com/conda-forge/argopy-feedstock/issues/15

Reply:
> NOTE: More recent versions of argopy pin `- gsw <=3.4.0`

I don't think this pin is needed in argopy b/c I don't believe we had any changes in gsw that could change how argopy works from 3.4 to 3.6 we have mostly bugfixes. Also,it is highly recommended to use gsw >=3.6 due to these bugfixes in the underlying C-library.

Reply:
until the feedstock is fixed we can get argopy from pip